app/javascript/elm/src/Main.elm

```diff
@@ -474,6 +506,18 @@ update msg model =
             ( { model | selectedSession = SelectedSession.updateRange model.selectedSession measurements }, Cmd.none )
 
 
+deselectSession : { a | selectedSession : WebData SelectedSession } -> ( { a | selectedSession : WebData SelectedSession }, Cmd Msg )
```

Comment:
The signature is a bit noisy. Consider either passing model or naming the type with an alias. If you go with the second consider renaming the argument to something different than `model`